android/build.gradle

```diff
@@ -2,11 +2,13 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 21
+        minSdkVersion = 24
         compileSdkVersion = 28
         targetSdkVersion = 28
         supportLibVersion = "28.0.0"
+        kotlinVersion = "1.3.0"
         glideVersion = "4.9.0"
+        googlePlayServicesVersion = "16.+"
```

Comment:
It seems you're not using this constant anywhere.

Reply:
it is used by some detox gradle dependencies (removing it fails build)